doc/lua_api.txt

```diff
@@ -7948,6 +7950,11 @@ Used by `HTTPApiTable.fetch` and `HTTPApiTable.fetch_async`.
         multipart = boolean
         -- Optional, if true performs a multipart HTTP request.
         -- Default is false.
+        -- Post only use post_data
+
+        post_data = "Raw POST request data string" OR {field1 = "data1", field2 = "data2"},
+        -- Deprecated, use only for multipart request
```

Comment:
so is this actually deprecated or do you still need to use this for POST requests?

Reply:
Only for the multipart requests.
All other requests can be done using method = "Get", data="somedata"

Reply:
Should rather use `method = "POST", data = string or table, type = "multipart" or "formdata"`

Reply:
@appgurueu 
Right, changed that so that multipart doesn't have to use the deprecated post_data